### ISSUE TITLE
Add metric to record the client was embedded.

### ISSUE THREAD

**Issue by hmstepanek** [hypothesis/client#1110]
It would be useful to track what pages have the client embedded on them. This is a placeholder to add a metric/some other monitoring solution to track that info.

**Comment by judell** [comment]
When a site embeds the H client, the client sends the site's domain in the (charmingly misspelled) http referer header. Once upon a time those headers showed up in http access logs that you could easily look through. Is that no longer the case? And if not, could we turn on that logging long enough to capture a few days' worth of /embed.js requests and associated headers? That would tell us a lot.

**Comment by hmstepanek** [comment]
I was just looking in the [cloudwatch](https://us-west-1.console.aws.amazon.com/cloudwatch/home?region=us-west-1#logEventViewer:group=/aws/elasticbeanstalk/h-prod/var/log/nginx/access.log;filter=referer%20-badge
) logs and I do see requests with the http referer header. I've made a dashboard in aws cloudwatch that in theory counts the number of requests in the access log that have said header. It doesn't appear to be working though and I'm not entirely sure what I'm doing wrong?? Might have to take a look at this again tomorrow.

**Comment by hmstepanek** [comment]
So I've added a dashboard to google analytics that tracks the number of events (these include changing groups, making/editing annotations, opening the sidebar, etc) for each category (embed, via, chrome-extension). I've also added a tabular list of the top referrer's that embed the client. I'll link to it/provide a screen shot in the metrics channel (https://hypothes-is.slack.com/archives/C0MJEF9FD/p1558533263000100). While google analytics does have all the referrer's it doesn't allow you to display more than the top 10 on one chart.
